src/Scrollbar.js

```diff
@@ -503,11 +503,11 @@ class Scrollbar extends React.PureComponent {
   }
 
   getTouchX = (/*object*/ e) => {
-    return Math.round(e.targetTouches[0].pageX - e.target.getBoundingClientRect().x);
+    return Math.round(e.targetTouches[0].clientX - e.target.getBoundingClientRect().x);
```

Comment:
So this fixes a small bug from #341, where clicking and dragging the scrollbar's divider is not in sync with the mouse when touch is used.